### DISCUSSION TITLE
Auto save feature

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#546] [Ideas]
This is like in the toliss and most recently the zibo. So when X-Plane crashes you can load in to the same place u were before the crash (don’t make it so it affects performance that much tho). It’s basically what it says on the tin.

**Comment by aeroniemi** [comment]
This requires custom FMS/CDU, although is on the agenda for after that occurs - is also a requirement for full shared flight compatability

**Reply by aeroniemi** [reply]
The tech to save our variables does already exist, although saving FMC components is a bit more difficult. Ifwe're happy to just not save the FMC then it could be done soon.

**Reply by Simleybob** [reply]
The FMC is crucial to a flight, so there’s not much point in making it a feature until you can save the fms.

Won’t not saving the fms, not allow you to save the position and location of the plane? And not to mention you would have to reprogram the whole flight route and performance mid air after a CTD for example.

**Reply by aeroniemi** [reply]
We can save position, so it would just be a case of re-importing and configuring your route, which isn't really too much effort, at least compared to the amount of effort of restarting your flight

**Reply by aeroniemi** [reply]
(it is also likely that you will never be able to save the route as we aren't planning to do custom route generation, so if that is a crucial feature for this then the FR can be marked as unlikely)

**Reply by Simleybob** [reply]
Alright, tbh with simbrief uplink on the 787 this could be a thing. I suppose as you can save the custom parts of the fmc - perf, irs and nav rad it would work.

let’s do it then!

**Reply by Simleybob** [reply]
What about a/p? Can you save that? 

I'm just a bit worried that if you use it, it will then be in the right place but the plane will kinda fall out the sky because a/p won't be engaged and that would be a nightmare.

**Reply by aeroniemi** [reply]
we'd probably set it so that it started paused with it configured to heading hold and 0 VS, which should keep it stable

**Reply by Simleybob** [reply]
Sounds good
